crates/astria-sequencer-utils/src/cli.rs

```diff
@@ -29,6 +34,10 @@ pub enum Command {
     /// Parse blob data from an arg, a file, or stdin
     #[command(arg_required_else_help = true)]
     ParseBlob(blob_parser::Args),
+
+    /// Estimate the activation point to schedule or check an upcoming upgrade
+    #[command(arg_required_else_help = true)]
+    EstimateActivationPoint(activation_point_estimator::Args),
```

Comment:
nit: I think this is slightly clearer
```suggestion
    /// Estimate the activation point of an upgrade to schedule or check an upcoming one
    #[command(arg_required_else_help = true)]
    EstimateActivationPoint(activation_point_estimator::Args),
```

Reply:
Done in [3a14b8a](https://github.com/astriaorg/astria/pull/2114/commits/3a14b8a236dbad758a6b54657bf21b17bbee2dbe).